config/routes.rb

```diff
@@ -354,6 +354,7 @@
           as: :in_person_ready_to_verify
       post '/in_person/usps_locations' => 'in_person/usps_locations#index'
       put '/in_person/usps_locations' => 'in_person/usps_locations#update'
+      post '/in_person/validate_transliterable' => 'in_person/validate_transliterable#index'
```

Comment:
I see we do this in other parts of the file but I would not `POST` to an `#index`, I associate `POST` with `#create` usually (even if we're not making a new record)

Reply:
I selected `POST` based on what similar API calls are doing. The request will contain PII and it's typical for `GET` requests (in many cases) to log query params. Let me know if you'd like to suggest a specific alternative HTTP method.

Reply:
... or I guess it sounds like you're suggesting that the controller use the `create` method instead?

Reply:
yeah I would switch to `#create` sorry